Lib/fontTools/misc/transform.py

```diff
@@ -346,6 +349,44 @@ def toPS(self):
         """
         return "[%s %s %s %s %s %s]" % self
 
+    def toDecomposed(self) -> "DecomposedTransform":
```

Comment:
Not sure if it's better, but I was thinking rather of a class method on DecomposedTransform:
```python
decomp = DecomposedTransform.fromTransform(t)
```
That way we can easily convert 6-tuples, too.

Reply:
I can add both?

Reply:
Yeah, why not.

Reply:
I also like the classmethod on DecomposedTransform

Reply:
we could even have both